dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/event/data/JdbcEnrollmentAnalyticsManager.java

```diff
@@ -63,6 +75,75 @@
     extends AbstractJdbcEventAnalyticsManager
         implements EnrollmentAnalyticsManager
 {
+    private static final Log log = LogFactory.getLog( JdbcEnrollmentAnalyticsManager.class );
+    
+    @Override
+    public Grid getEnrollments(EventQueryParams params, Grid grid, int maxLimit) 
+    {
+        List<String> fixedCols = Lists.newArrayList( "pi", "tei", "enrollmentdate", "ST_AsGeoJSON(pigeometry)", "longitude", "latitude", "ouname", "oucode" );
+
+        List<String> selectCols = ListUtils.distinctUnion( fixedCols, getSelectColumns( params ) );
+
+        String sql = "select " + StringUtils.join( selectCols, "," ) + " ";
+
+        sql += getFromClause( params );
+
+        sql += getWhereClause( params );
+
+        //sql += getSortClause( params );
```

Comment:
Remember to remove these comments

Reply:
Yes, actually they where not implemented but will be...